.github/workflows/dummy-agent-test.yml

```
@@ -0,0 +1,21 @@
+name: Run e2e test with dummy agent
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

nit (to keep it consistent with other CI actions):

```suggestion
        uses: actions/setup-python@v5
```